starcheck/src/lib/Ska/Parse_CM_File.pm

```diff
@@ -479,8 +487,8 @@ sub DOT {
 
     foreach (keys %command) {
         %{$dot{$_}} = parse_params($command{$_});
-        $dot{$_}{time}  = date2time($dot{$_}{TIME})     if ($dot{$_}{TIME});
-        $dot{$_}{time} += date2time($dot{$_}{MANSTART}) if ($dot{$_}{TIME} && $dot{$_}{MANSTART});
+        $dot{$_}{time}  = date2time($dot{$_}{TIME}) if ($dot{$_}{TIME});
+        $dot{$_}{time} += rel_date2time($dot{$_}{MANSTART}) if ($dot{$_}{TIME} && $dot{$_}{MANSTART});
```

Comment:
I should add a comment here about why the MANSTART time needs the relative datetime method, and should add a comment to the newish rel_date2time method indicating what time formats it expects.